src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -180,36 +181,40 @@ int main(int argc, char** argv)
       ("matchingEstimator", po::value<robust::EROBUST_ESTIMATOR>(&matchingEstimator)->default_value(matchingEstimator), 
           std::string("The type of *sac framework to use for matching "
           "{"+str_estimatorChoices+"}").c_str())
-        ("sfmdata", po::value<std::string>(&sfmFilePath)->required(),
-            "The sfm_data.json kind of file generated by OpenMVG.")
-        ("descriptorPath", po::value<std::string>(&descriptorsFolder)->required(),
-            "Folder containing the .desc.")
-        ("mediapath", po::value<std::string>(&mediaPath)->required(),
-            "The folder path containing all the synchronised image subfolders "
-            "assocated to each camera")
-        ("filelist", po::value<std::string>(&filelist),
-            "An optional txt file containing the images to use for calibration. "
-            "This file must have the same name in each camera folder and contains "
-            "the list of images to load.")
-        ("refineIntrinsics", po::bool_switch(&refineIntrinsics),
-            "Enable/Disable camera intrinsics refinement for each localized image")
-        ("nCameras", po::value<size_t>(&numCameras)->default_value(numCameras), 
+      ("sfmdata", po::value<std::string>(&sfmFilePath)->required(),
+          "The sfm_data.json kind of file generated by OpenMVG.")
+      ("descriptorPath", po::value<std::string>(&descriptorsFolder)->required(),
+          "Folder containing the .desc.")
+      ("mediapath", po::value<std::string>(&mediaPath)->required(),
+          "The folder path containing all the synchronised image subfolders "
+          "assocated to each camera")
+      ("filelist", po::value<std::string>(&filelist),
+          "An optional txt file containing the images to use for calibration. "
+          "This file must have the same name in each camera folder and contains "
+          "the list of images to load.")
+      ("refineIntrinsics", po::bool_switch(&refineIntrinsics),
+          "Enable/Disable camera intrinsics refinement for each localized image")
+      ("nCameras", po::value<size_t>(&numCameras)->default_value(numCameras), 
           "Number of cameras composing the rig")
-        ("calibration", po::value<std::string>(&rigCalibPath)->required(), 
+      ("calibration", po::value<std::string>(&rigCalibPath)->required(), 
           "The file containing the calibration data for the rig (subposes)")
-        ("reprojectionError", po::value<double>(&resectionErrorMax)->default_value(resectionErrorMax), 
+      ("reprojectionError", po::value<double>(&resectionErrorMax)->default_value(resectionErrorMax), 
           "Maximum reprojection error (in pixels) allowed for resectioning. If set "
           "to 0 it lets the ACRansac select an optimal value.")
+      ("useLocalizeRigNaive", po::bool_switch(&useLocalizeRigNaive),
```

Comment:
It would be more explicit to have `--rigLocalizationMethod` with a default value of "opengv" if builded with or "naive" if not.